### ISSUE TITLE
Cannot parse scss variables with "url" in the name with a comment following

### ISSUE THREAD

**Issue by dane-harnett** [tonyganch/gonzales-pe#95]
Parse error occurs on the following snippet:

```
$a-url: 'myUrl.png';
// comment

```

**Comment by DanPurdy** [comment]
Hi there, 
I'm also seeing something similar. Currently seeing some strange parse error with certain selector names when comments are present within a block

``` scss
.url-overflow {
  // this is a comment
  content: '';
}
```

Interestingly no parse error is present for the following:

``` scss

// no - in selector name
.urloverflow {
  // this is a comment
  content: '';
}

//no use of url in selector name
.test-overflow {
  // this is a comment
  content: '';
}

//no comment within block
.url-overflow {
  content: '';
}
```

The only thing having changed is the name of the selector removing a `-` though if the selector doesn't contain the word url with that symbol it acts perfectly fine.

The error is always reporting on the line with the comment though and removing the comment from the block also reports no parse error.

This was originally reported here
https://github.com/sasstools/sass-lint/issues/229

If you need any more information let me know.

Thanks!

**Comment by tonyganch** [comment]
@dane-harnett @DanPurdy, I think the issue has been fixed with #104.
I've added tests for both of your examples above and they pass.
Please check `dev` branch or `@3.0.2` version and let me know if you find that the issue is still there.

**Comment by DanPurdy** [comment]
Hi @tonyganch I've just run the original issue and the code I posted above against 3.2.1 and all is fine. Thanks!
